### PR TITLE
Improve logging in sbrRequestMapper

### PR DESCRIPTION
**Pull request by isutton**

### CODE REVIEW

File: pkg/controller/servicebindingrequest/mapper.go [hunk 1]
```diff
@@ -132,6 +133,8 @@ func (m *sbrRequestMapper) Map(obj handler.MapObject) []reconcile.Request {
 		}
 		log.Debug("current resource is a SBR", "Requests", requests)
 		return requests
+	} else {
```
Comment (Avni-Sharma) [inline pkg/controller/servicebindingrequest/mapper.go:136]: else is redundant here

Comment (isutton) [reply pkg/controller/servicebindingrequest/mapper.go:136]: You're right in principle, but in order to make it easier to follow the debug logs more information should be added.

**Comment by pedjak** [comment]
obsolete after #919 got merged.

**Pull request closed without merging**